src/ansys/aedt/toolkits/antenna/backend/models/conical_spiral.py

```diff
@@ -294,12 +297,12 @@ def model_hfss(self):
 
     @pyaedt_function_handler()
     def model_disco(self):
-        """Model in PyDisco. To be implemented."""
+        """Model in PyDiscovery. To be implemented."""
```

Comment:
@Samuelopez-ansys I know that all the mentions of "PyDiscovery" say "To be implemented." However, I only hear talk these days of PyAnsys Geometry. Is PyDiscovery something still being worked on?